specification/communication/data-plane/Chat/stable/2023-11-07/communicationserviceschat.json

```diff
@@ -0,0 +1,1793 @@
+          }
+        }
+      }
+    },
+    "/chat/threads/{chatThreadId}/typing": {
+      "post": {
+        "tags": [
+          "TypingNotifications"
+        ],
+        "summary": "Posts a typing event to a thread, on behalf of a user.",
+        "operationId": "ChatThread_SendTypingNotification",
+        "consumes": [
+          "application/json",
+          "application/merge-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "Id of the thread.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "sendTypingNotificationRequest",
+            "description": "Details of the typing notification request.",
+            "schema": {
+              "$ref": "#/definitions/SendTypingNotificationRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful."
+          },
+          "401": {
+            "description": "Unauthorized.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Forbidden.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Post typing event to a thread": {
+            "$ref": "./examples/Messages_SendTypingNotification.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationUserIdentifierModel": {
+      "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationUserIdentifierModel"
+    },
+    "CommunicationCloudEnvironment": {
+      "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationCloudEnvironmentModel"
+    },
+    "MicrosoftTeamsUserIdentifierModel": {
+      "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/MicrosoftTeamsUserIdentifierModel"
+    },
+    "CommunicationIdentifierModel": {
+      "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationIdentifierModel"
+    },
```

Comment:
In 2023-07-01-preview, these refs all used the more recent `Common/stable/2022-07-13` versions of these definitions. Why did you decide to go back to using the older ones?

Reply:
That's a good catch, thank you was copying the folder from previous stable.